src/v2/Apps/ViewingRoom/viewingRoomRoutes.tsx

```diff
@@ -92,6 +93,12 @@ export const viewingRoomRoutes: AppRouteConfig[] = [
       },
       {
         path: "works",
+        render: () => {
+          throw new RedirectException("artworks", 301)
+        },
+      },
+      {
+        path: "artworks",
         Component: WorksRoute,
```

Comment:
**question**: Are we able to remove some of the other properties/queries on this route if we're always responding with a 301 redirect?

Reply:
@dblandin I’m not sure I follow this question, there is no other properties for `/works` route
https://github.com/artsy/force/blob/9db09af6e8c4d4676b28f1afb589cbb178a8d6af/src/v2/Apps/ViewingRoom/viewingRoomRoutes.tsx#L94-L99

Reply:
Oh sorry, you're totally right. I was looking down at the next block and thinking it was part of `/works`. Never mind!